Jenkinsfile-centos-sgx-sim

```diff
@@ -0,0 +1,139 @@
+#!groovy
```

Comment:
As per docker-compose file naming convention, sgx suffix indicates HW mode. To keep naming conventions consitent, can we rename the file to Jenkinsfile-centos-sim?